adapters/repos/db/file_structure_migration.go

```diff
@@ -118,3 +132,78 @@ func makeNewRelPath(oldPath, prefix string) (newPath string) {
 	}
 	return
 }
+
+func (db *DB) migratePQVectors(plan migrationPlan) error {
+	for _, parts := range plan {
+		oldClassPath := path.Join(
+			db.config.RootPath,
+			toTitleCase(parts[0].index),
+		)
+		oldStorePath := path.Join(
+			oldClassPath,
+			parts[0].shard,
+			"compressed_objects",
+		)
+
+		exists, err := fileExists(oldStorePath)
+		if err != nil {
+			return fmt.Errorf("check store exists: %w", err)
+		}
+
+		if !exists {
+			continue
+		}
+
+		newStorePath := path.Join(
+			oldClassPath,
```

Comment:
if old path is used to build a new one, does it mean that if old one had class/index name with uppercases, new one will also have uppercases? shouldn't be lowercased?

Reply:
You're right, a different approach is needed. Latest commit handles PQ migration differently